packages/governance/src/singlePluralityVoteCounter.js

```diff
@@ -0,0 +1,223 @@
+// @ts-check
+
+import { keyEQ, makeHeapFarInstance, makeStore } from '@agoric/store';
+import { E } from '@endo/eventual-send';
+import { makePromiseKit } from '@endo/promise-kit';
+import {
+  buildQuestion,
+  ChoiceMethod,
+  coerceQuestionSpec,
+  ElectionType,
+  positionIncluded,
+} from './question.js';
+import { scheduleClose } from './closingRule.js';
+import {
+  VoteCounterAdminI,
+  VoteCounterCloseI,
+  VoteCounterPublicI,
+} from './typeGuards.js';
+import { makeQuorumCounter } from './quorumCounter.js';
+
+const { details: X } = assert;
+
+const validateQuestionSpec = questionSpec => {
+  coerceQuestionSpec(questionSpec);
+
+  questionSpec.electionType === ElectionType.ELECTION ||
+    questionSpec.electionType === ElectionType.SURVEY ||
+    assert.fail(X`${questionSpec.electionType} must be ELECTION or SURVEY`);
+
+  questionSpec.maxChoices === 1 ||
+    assert.fail(X`Can only choose 1 item on a multiple question`);
+
+  questionSpec.method === ChoiceMethod.PLURALITY ||
+    assert.fail(X`${questionSpec.method} must be PLURALITY`);
+};
+
+/** @type {BuildVoteCounter} */
+const makeSinglePluralityVoteCounter = (
```

Comment:
This doesn't need to be a separate voteCounter. Everything it does can be done with `multiPluralityVoteCounter`, with MaxChoices set to 1. Please drop this class and update the tests to rely on `multiPluralityVoteCounter`, which could be renamed to `multiCandidateVoteCounter`.